source/compatibility.tex

```diff
@@ -2411,8 +2409,8 @@
 with a type.
 In \Cpp{}, class members can be declared with the \keyword{static} storage
 class specifier.
-Allowing storage class specifiers on type
-declarations could render the code confusing for users.
+Storage class specifiers on type
+declarations can render the code confusing for users.
```

Comment:
Maybe just "can be confusing", since we're now talking about the specifiers themselves rather than allowing them?